src/it/it-source-failed/src/test/java/name/valery1707/kaitai/it/KaitaiIcoTest.java

```diff
@@ -0,0 +1,49 @@
+package name.valery1707.kaitai.it;
+
+import io.kaitai.struct.ByteBufferKaitaiStream;
+import io.kaitai.struct.KaitaiStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.Test;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'org\.junit\.Test' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.